packages/volto/src/components/manage/Aliases/Aliases.jsx

```diff
@@ -19,344 +15,262 @@ import {
+              primary
+              disabled={aliasesToRemove.length === 0}
+            >
+              <FormattedMessage id="Remove" defaultMessage="Remove" />
+            </Button>
+          </Segment>
+        </Form>
+      </Segment.Group>
+      {isClient &&
+        createPortal(
+          <Toolbar
+            pathname={pathname}
+            hideDefaultViewButtons
+            inner={
+              <Link to={`${getBaseUrl(pathname)}`} className="item">
+                <Icon
+                  name={backSVG}
+                  className="contents circled"
+                  size="30px"
+                  title={intl.formatMessage(messages.back)}
                 />
-              </Header>
-              {this.props.aliases?.items.map((alias, i) => (
-                <Form.Field key={i}>
-                  <Checkbox
-                    id={`alias-check-${i}`}
-                    onChange={(e, { value }) => this.handleCheckAlias(value)}
-                    value={alias.path}
-                    label={alias.path}
-                    checked={this.state.aliasesToRemove.includes(alias.path)}
-                  />
-                </Form.Field>
-              ))}
-              <Button
-                id="remove-alias"
-                onClick={() => this.handleRemoveAliases()}
-                primary
-                disabled={this.state.aliasesToRemove.length === 0}
-              >
-                <FormattedMessage id="Remove" defaultMessage="Remove" />
-              </Button>
-            </Segment>
-          </Form>
-        </Segment.Group>
-        {this.state.isClient &&
-          createPortal(
-            <Toolbar
-              pathname={this.props.pathname}
-              hideDefaultViewButtons
-              inner={
-                <Link
-                  to={`${getBaseUrl(this.props.pathname)}`}
-                  className="item"
-                >
-                  <Icon
-                    name={backSVG}
-                    className="contents circled"
-                    size="30px"
-                    title={this.props.intl.formatMessage(messages.back)}
-                  />
-                </Link>
-              }
-            />,
-            document.getElementById('toolbar'),
-          )}
-      </Container>
-    );
-  }
-}
+              </Link>
+            }
+          />,
+          document.getElementById('toolbar'),
+        )}
+    </Container>
+  );
+};
+
+Aliases.propTypes = {
+  removeAliases: PropTypes.func.isRequired,
+  addAliases: PropTypes.func.isRequired,
+  getAliases: PropTypes.func.isRequired,
+  pathname: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+};
+
+const mapStateToProps = (state, props) => ({
+  aliases: state.aliases,
+  pathname: props.location.pathname,
+  title: state.content.data?.title || '',
+});
+
+const mapDispatchToProps = {
+  addAliases,
+  getAliases,
+  removeAliases,
+  getContent,
+};
```

Comment:
The functional component should use getSelector, getLocation, and getDispatch, instead of using `connect`

Reply:
I'll close this to reduce noise in the list of PRs for review. If you update it, you can ask us to reopen it.